lib/cache.js

```diff
@@ -49,7 +65,7 @@ module.exports = (log, config, namespace) => {
     },
 
     /**
-     * Fetch data from the cache, keyed by a string.
+     * Set data in the cache, keyed by a string.
```

Comment:
Might be worthwhile to put a quick note here that this will override existing data and we should use `add` if we don't want that.

Reply:
I am probably missing something, but it doesn't appear that we use the `.set` method anywhere in the code? The initial metricsContext is set in the content-server and will only ever be updated here?

Reply:
Yeah, it's not used any more. Happy to remove it if you like.

Reply:
@philbooth While definately not optimal to keep code that isn't being used, I think in this case it might be ok? It kind of completes the cache implementation. I'll defer to you :)